rudder-server-relay/SPECS/rudder-server-relay.spec

```diff
@@ -93,7 +93,11 @@ Requires: python3, python3-mod_wsgi
 BuildRequires: python
 Requires: python, apache2-mod_wsgi, python-pyOpenSSL
 %endif
-%if 0%{?suse_version} && 0%{?suse_version} >= 1500
+%if 0%{?suse_version} && 0%{?suse_version} == 1500
+BuildRequires: python
+Requires: python, apache2-mod_wsgi-python
```

Comment:
do you mean we will use python2

Reply:
yes, there is no apache2-mod_wsgi-python3 support on sles15